chainer/link.py

```diff
@@ -315,8 +315,9 @@ def zerograds(self):
         iteration of the optimization.
 
         """
-        for param in self.params():
-            param.zerograd()
```

Comment:
It seems better to remove `Chain.zerograds` and `ChainList.zerograds` than removing the above statement.